sdk/servicebus/service-bus/test/public/utils/testutils2.ts

```diff
@@ -173,13 +173,24 @@ export function getRandomTestClientTypeWithNoSessions(): TestClientType {
  * Returns a TestClientType for either a Queue or a Subscription with
  * sessions enabled
  */
-export function getRandomTestClientTypeWithSessions(): TestClientType {
-  const withSessionTestClientTypes = [
-    TestClientType.PartitionedQueueWithSessions,
-    TestClientType.PartitionedSubscriptionWithSessions,
-    TestClientType.UnpartitionedQueueWithSessions,
-    TestClientType.UnpartitionedSubscriptionWithSessions
-  ];
+export function getRandomTestClientTypeWithSessions(
+  type: "all" | "subscription" | "queue" = "all"
```

Comment:
Nit: Should this be `any` instead of `all`?

Reply:
I chose 'all' since 'any' is....well, already a type. :)